hooks/VDM.js

```diff
@@ -305,303 +305,229 @@ module.exports = function (input, session) {
   }
 
   if (data.waveperiod) {
     contextPrefix = 'meteo.'
-    if (data.waveperiod < 61) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.wavePeriod`,
-        value: data.waveperiod,
-      })
-    }
+    values.push({
+      path: `environment.water.wavePeriod`,
+      value: data.waveperiod,
+    })
   }
 
   if (data.wavedir) {
     contextPrefix = 'meteo.'
-    if (data.wavedir < 360) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.waveDirection`,
-        value: utils.transform(data.wavedir, 'deg', 'rad'),
-      })
-    }
+    values.push({
+      path: `environment.water.waveDirection`,
+      value: utils.transform(data.wavedir, 'deg', 'rad'),
+    })
   }
 
   if (data.swellhgt) {
     contextPrefix = 'meteo.'
-    if (data.swellhgt < 252) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.swellHeight`,
-        value: data.swellhgt / 10,
-      })
-    }
+    values.push({
+      path: `environment.water.swellHeight`,
+      value: data.swellhgt,
+    })
   }
 
   if (data.swellperiod) {
     contextPrefix = 'meteo.'
-    if (data.swellperiod < 61) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.swellPeriod`,
-        value: data.swellperiod,
-      })
-    }
+    values.push({
+      path: `environment.water.swellPeriod`,
+      value: data.swellperiod,
+    })
   }
 
   if (data.swelldir) {
     contextPrefix = 'meteo.'
-    if (data.swelldir < 360) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.swellDirection`,
-        value: utils.transform(data.swelldir, 'deg', 'rad'),
-      })
-    }
+    values.push({
+      path: `environment.water.swellDirection`,
+      value: utils.transform(data.swelldir, 'deg', 'rad'),
+    })
   }
 
   if (data.seastate) {
     contextPrefix = 'meteo.'
-    if (data.seastate < 13) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.seaState`,
-        value: beaufortScale[data.seastate],
-      })
-    }
+    values.push({
+      path: `environment.water.seaState`,
+      value: beaufortScale[data.seastate],
+    })
   }
 
   if (data.watertemp) {
     contextPrefix = 'meteo.'
-    if (data.watertemp > -101 && data.watertemp < 501) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.temperature`,
-        value: utils.transform((data.watertemp / 10), 'c', 'k'),
-      })
-    }
+    values.push({
+      path: `environment.water.temperature`,
+      value: utils.transform(data.watertemp, 'c', 'k'),
+    })
   }
 
   if (data.precipitation) {
     contextPrefix = 'meteo.'
-    if (data.precipitation < 7) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.precipitation`,
-        value: precipitationType[data.precipitation],
-      })
-    }
+    values.push({
+      path: `environment.air.precipitation`,
```

Comment:
For values of 'rain', 'snow', etc should this be `environment.observations.outside.precipitationType`?

Forecasts will often contain the volume of precipitation so to cater for this scenario maybe use paths like
`precipitationType` & `precipitationVolume`.